File: src/aws/lambdas/incremental_distribution/src/test/java/uk/nhs/nhsx/core/aws/secretsmanager/SecretValueTest.java
package uk.nhs.nhsx.core.aws.secretsmanager;

import org.junit.Test;

import static org.hamcrest.CoreMatchers.containsString;
import static org.hamcrest.CoreMatchers.not;
import static org.hamcrest.MatcherAssert.assertThat;

public class SecretValueTest {

    @Test
    public void toStringDoesntGiveAnythingAway() throws Exception {
        assertThat(SecretValue.of("something").toString(), not(containsString("something")));
    }
}

File: src/aws/lambdas/incremental_distribution/src/test/java/uk/nhs/nhsx/core/aws/secretsmanager/SecretValueTest.kt
package uk.nhs.nhsx.core.aws.secretsmanager

import org.hamcrest.CoreMatchers.containsString
import org.hamcrest.CoreMatchers.not
import org.hamcrest.MatcherAssert.assertThat
import org.junit.jupiter.api.Test

class SecretValueTest {

    @Test
    fun toStringDoesntGiveAnythingAway() {
        assertThat(SecretValue.of("something").toString(), not(containsString("something")))
    }
}